src/pages/enterprise/auth-connect.md

```diff
@@ -589,6 +715,37 @@ ___
 
 
 
+### [1.3.3] (2019-11-08)
+
+
+### Bug Fixes
+
+* **ios:** rename AFNetworking symbols that conflic with advanced-http plugin 
+
+
+
+### [1.3.2] (2019-11-01)
+
+
+
+### [1.3.1] (2019-10-29)
+
+
+
+### [1.3.0] (2019-10-09)
```

Comment:
details? Via Luke suggestion, `fix an issue to remove dependency on IdentityVault`?